frontend/src/scenes/pipeline/utils.tsx

```diff
@@ -369,26 +372,27 @@ export function pipelinePluginBackedNodeMenuCommonItems(
                               callback: loadPluginConfigs,
                           })
                       },
-                      disabledReason: canConfigurePlugins ? undefined : 'You do not have permission to delete.',
+                      disabledReason: notAllowedReasonByStageAndOperationType[node.stage]['edit_without_enable'],
                   },
               ]
             : []),
     ]
 }
 
-export function checkPermissions(stage: PipelineStage, togglingToEnabledOrNew: boolean): boolean {
-    const { canConfigurePlugins, canEnableNewDestinations } = useValues(pipelineLogic)
```

Comment:
I suspect this is the reason why updates started failing, we can't call a function with using from a logic